Assets/Scripts/WorldEngine/World.cs

```diff
@@ -155,9 +155,9 @@ public class World : ISynchronizable
     public const float Circumference = 40075; // In kilometers;
 
     //public const int NumContinents = 7;
-    public const int NumContinents = 12;
-    //public const float ContinentBaseWidthFactor = 0.8f;
-    public const float ContinentBaseWidthFactor = 1.1f;
+    public const int NumContinents = 15;
```

Comment:
Why the increase to 15?

Reply:
Those are just tweaks to make continental landmasses a little bit bigger and more interesting. After the changes I made to add rivers I felt most landmasses ended up looking like smallish rounded blobs. Tough now I might have gone too much in the opposite direction. Let me know if you feel continents looked better before this change.